app/views/monitoring-stations/view.njk

```diff
@@ -0,0 +1,150 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% macro formatCell(licence, licenceDetail) %}
+  {% for linkage in licence.linkages %}
+    <p class="govuk-!-margin-bottom-1 govuk-body-m"> {{ linkage[licenceDetail] }}</p>
+  {% endfor %}
+{% endmacro %}
+
+{% block breadcrumbs %}
+  {# Back link #}
+  {{
+    govukBackLink({
+      text: 'Back',
+      href: "/licences"
+    })
+  }}
+{% endblock %}
+
+{% block content %}
+  {# Main heading #}
+  <div class="govuk-body">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">
+      {{ pageTitle }}
+    </h1>
+  </div>
+
+  <div class="govuk-!-margin-bottom-7">
+    {{ govukSummaryList({
+      classes: 'govuk-summary-list--no-border',
+      attributes: {
+        'data-test': 'meta-data'
+      },
+      rows: [
+        {
+          key: { text: "Grid reference", classes: "meta-data__label" },
+          value: { html: '<span data-test="meta-data-grid-region">' + gridReference + '</span>', classes: "meta-data__value" }
+        },
+        {
+          key: { text: "WISKI ID", classes: "meta-data__label" },
+          value: { html: '<span data-test="meta-data-wiski-id">' + wiskiId + '</span>', classes: "meta-data__value" }
+        },
+        {
+          key: { text: "Station reference", classes: "meta-data__label" },
+          value: { html: '<span data-test="meta-data-station-reference">' + stationReference + '</span>', classes: "meta-data__value" }
+        }
+      ]
+    }) }}
+  </div>
+
+  {% if licences.length > 0 and hasPermissionToSendAlerts %}
```

Comment:
I just want to double check (I really can't remember!)

But is this correct? You have to have permissions to send alerts to see the button _and_ the table?

I would of assumed it was just the button that was hidden from those without the permission.

Reply:
I had another look, and I implemented the logic wrong. The number of if statements was quite confusing. I have now separated it out into the correct conditions!